ui/admin/tests/e2e/helpers/vault-cli.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-undef */
+const { execSync } = require('child_process');
+
+/**
+ * Checks that the vault cli is available
+ */
+exports.checkVaultCli = async () => {
```

Comment:
Does this means we need vault install to run the test? or just the CLI presence?

Reply:
As currently set up, this test needs both the Vault CLI and a Vault server instance running.